framework/CHANGELOG.md

```diff
@@ -9,12 +9,13 @@ Yii Framework 2 Change Log
 - Enh #12938: Allow to pass additional parameters to `yii\base\View::renderDynamic()` (mikehaertl)
 - Enh #13006: Added a `/` to the `yii\captcha\Captcha::$captchaAction` string to work correctly in a module also (boehsermoe)
 - Removed methods marked as deprected in 2.0.x (samdark)
+- Chg #7770: Updated the fallback date formats used when no `intl` extension is installed to match the defaults of the latest ICU version (cebe)
```

Comment:
@samdark actually if devs would put their issues into the correct order right away we would not have the weird reordering issue on merge :)